src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -76,7 +78,7 @@ object DeployPlugin extends AutoPlugin {
       */
     val deployDirs = settingKey[Seq[String]](
       "subdirectories from the stage task to copy during deploy. " +
-        "defaults to bin/, conf/, lib/, and public/"
```

Comment:
I would not change this - it's quite possible we're relying on other items in `conf/` being copied over.